meson.build

```
@@ -159,8 +159,9 @@ if not get_option('source-only')
     sdl_dep = dependency('sdl2', fallback: ['sdl2', 'sdl2_dep'],
         default_options: default_fallback_options + sdl_options
     )
+    harfbuzz_dep = dependency('harfbuzz', fallback: ['harfbuzz'])
```

```suggestion

    harfbuzz_options = ['default_library=static']

    harfbuzz_options += 'glib=disabled'
    harfbuzz_options += 'gobject=disabled'
    harfbuzz_options += 'cairo=disabled'
    harfbuzz_options += 'icu=disabled'
    harfbuzz_options += 'freetype=enabled'
    harfbuzz_options += 'tests=disabled'
    harfbuzz_options += 'introspection=disabled'
    harfbuzz_options += 'docs=disabled'
    harfbuzz_options += 'doc_tests=false'
    harfbuzz_options += 'utilities=disabled'

    harfbuzz_dep = dependency('harfbuzz', fallback: ['harfbuzz'],
        default_options: default_fallback_options + harfbuzz_options
    )
```
Those options should get it properly building on the Linux and MINGW CI